.github/workflows/build-test.yml

```diff
@@ -582,6 +582,19 @@ jobs:
         id: test_notebook_vscode_ubuntu
         if: matrix.python != 'noPython' && matrix.os == 'ubuntu-latest' && matrix.webOrDesktop != 'web'
 
+      - name: Get server launcher Linux/macOS
+        if: ${{ github.event.inputs.os != 'windows' }}
+        run: |
+          wget -O- https://aka.ms/install-vscode-server/setup.sh | sh
+
+      - name: Start server distro with vscode.dev
+        if: always()
```

Comment:
Will change this to `failure()` once validated it's working properly.

Reply:
Doesn't this allow anyone to view the environment variables and other secrets?